test/src/test/java/hudson/model/ItemsTest.java

```diff
@@ -242,7 +242,7 @@ private void cannotOverwrite(String target) throws Exception {
                         .withRedirectEnabled(false)
                         .withThrowExceptionOnFailingStatusCode(false);
                 WebResponse webResponse = wc.getPage(new WebRequest(new URL(wc.getContextPath() + "createItem?name=" + target + "&mode=hudson.model.FreeStyleProject"), HttpMethod.POST)).getWebResponse();
-                if (webResponse.getStatusCode() != HttpStatus.SC_MOVED_TEMPORARILY) {
+                if (webResponse.getStatusCode() != HttpURLConnection.HTTP_MOVED_TEMP) {
```

Comment:
What would be the motivation behind this?

Reply:
HttpStatus is deprecated, so I chose the newer version from java.net